operator/controllers/helmrelease_controller_test.go

```diff
@@ -151,8 +151,14 @@ var _ = Describe("HelmReleaseReconciler", func() {
 			By("reconciling a failed updated release")
 			// TODO
 
-			By("reconciling a release with the deletion timestamp set ")
-			// TODO
+			By("reconciling a release that has been deleted")
```

Comment:
I don't really like that this test depends on the previous one, but I guess this is testing the full lifecycle.

Reply:
The alternative would be a sequence of tests like

* install success/fail
* install success, update success/fail
* install success, update fail, rollback success/fail

etc, each test case has to set up more and more initial state. I didn't think that would scale well, so I went for a full lifecycle single test case.

Reply:
Yeah I'm okay with this for now.